### PR TITLE
added request param descriptions and added requirements.txt

### LINKED ISSUE #10: Add request param descriptions and update Swagger models help text
Example in PR: 
https://github.com/IBM/MAX-Object-Detector/pull/20

### LINKED ISSUE #12: pip install from requirements.txt
Example PR:
https://github.com/IBM/MAX-Review-Text-Generator/pull/7

### PR DESCRIPTION
**Pull request by bdwyer2**
resolves #10, resolves #12 

This PR also simplifies the swagger output. See https://github.com/IBM/MAX-News-Text-Generator/issues/16

### CODE REVIEW

File: api/model.py [hunk 1]
```diff
@@ -26,18 +23,16 @@ def get(self):
...
         return MODEL_META_DATA
 
-label_prediction = api.model('LabelPrediction', {
-    'embedding': fields.List(fields.List(fields.Float,required=True))
-})
 
 predict_response = api.model('ModelPredictResponse', {
     'status': fields.String(required=True, description='Response status message'),
-    'predictions': fields.List(fields.Nested(label_prediction), description='Predicted labels and probabilities')
+    'embedding': fields.List(fields.List(fields.Float, required=True, description="Generated embedding"))
```
Comment (xwu0226) [inline api/model.py:29]: Will everyone know what `embedding` means here?

Comment (bdwyer2) [reply api/model.py:29]: I think so. The name of the model is Audio Embedding Generator

File: tests/test.py [hunk 2]
```diff
@@ -17,11 +17,11 @@ def test_response():
...
-    assert len(response['predictions'][0]['embedding']) == 4
-    assert len(response['predictions'][0]['embedding'][0]) == 128
-    assert len(response['predictions'][0]['embedding'][1]) == 128
-    assert len(response['predictions'][0]['embedding'][2]) == 128
-    assert len(response['predictions'][0]['embedding'][3]) == 128
+    assert len(response['embedding']) == 4
+    assert len(response['embedding'][0]) == 128
+    assert len(response['embedding'][1]) == 128
+    assert len(response['embedding'][2]) == 128
+    assert len(response['embedding'][3]) == 128
```
Comment (xwu0226) [inline tests/test.py:24]: each `embedding` is a list of float?

Comment (bdwyer2) [reply tests/test.py:24]: Yep, each embedding is a list of floats.

Eg
```
{
  "status": "ok",
  "result": [
    [
      158,
      23,
      150,
      ...
    ],
    ...,
    ...,
    [
      163,
      29,
      178,
      ...
    ]
  ]
}
```

Comment (bdwyer2) [reply tests/test.py:24]: I guess I should update that example output while I'm at it

**Review by MLnick — COMMENTED** [review]
Changes look good, just the example output in README needs to be updated to reflect

**Comment by bdwyer2** [comment]
@MLnick I think the README was incorrect before. The current output from this PR matches the README.

**Comment by MLnick** [comment]
@bdwyer2 ah right I see, missed that. LGTM

**Review by MLnick — APPROVED** [review]

**Pull request merged**